src/test/java/gov/cms/madie/measure/utilities/qicore411/PackagingUtilityImplTest.java

```diff
@@ -127,4 +109,43 @@ void testGetZipBundleWithLibraries() throws IOException {
             .startsWith("{\n  \"resourceType\": \"Library\","),
         is(true));
   }
+
+  @Test
+  void testGetZipBundleForTestCases() throws IOException {
+    PackagingUtility utility = new PackagingUtilityImpl();
+    String testCaseBundleJson = getStringFromTestResource("/testCaseBundle.json");
+    Bundle testCaseBundle =
+        FhirContext.forR4().newJsonParser().parseResource(Bundle.class, testCaseBundleJson);
+    byte[] tc1 = utility.getZipBundle(testCaseBundle, "TC1");
+    assertNotNull(tc1);
+
+    Map<String, String> zipContents = new HashMap<>();
+    getFileContents(tc1, zipContents);
+    assertThat(zipContents.size(), is(1));
+    assertThat(zipContents.containsKey("TC1.json"), is(true));
+  }
+
+  private void getFileContents(byte[] inputBytes, Map<String, String> zipContents)
```

Comment:
could this be refactored to return the Map<> zipContents rather than mutating the input param